### PR TITLE
feat(styles): new horizon spec for link [ci visual]

### LINKED ISSUE #4379: Component Design Update: Link
- [ ] Horizon specs(visual and interaction) updates
- [ ] Delta theming
- [ ] identify and document breaking changes

### PR DESCRIPTION
**Pull request by droshev**
new horizon spec for link + add link states

fixes #4379

### CODE REVIEW

File: packages/styles/src/link.scss [hunk 1]
```diff
@@ -12,6 +12,10 @@ $block: #{$fd-namespace}-link;
 
   @include fd-icon-selector() {
     color: var(--sapLinkColor);
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (InnaAtanasova) [inline packages/styles/src/link.scss:17]: @include fd-flex-center()

File: packages/styles/src/textarea.scss [hunk 2]
```diff
@@ -24,6 +24,15 @@ $block: #{$fd-namespace}-textarea;
...
   cursor: text;
 
+  // &[readonly],
+  // &.is-readonly {
+  //   font-size: var(--sapFontSmallSize);
+  //   color: var(--sapField_TextColor);
+  //   border-radius: var(--sapField_BorderCornerRadius);
+  //   background-color: var(--sapField_ReadOnly_Background);
+
+  // }
```
Comment (InnaAtanasova) [inline packages/styles/src/textarea.scss:34]: if we don't use maybe remove? Or if we decide to keep add comments on why it's commented out.

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: packages/styles/src/link.scss (hunk 1 above)
Comment (droshev) [reply packages/styles/src/link.scss]: done

File: packages/styles/src/mixins/_mixins.scss [hunk 3]
```diff
@@ -201,12 +201,13 @@
   align-items: center;
   transition: all $fd-animation-speed ease-in;
   line-height: inherit;
-  padding: 0 0.0625rem;
+  border-color: transparent;
```
[the remark this answers is not in the document]
Comment (droshev) [reply packages/styles/src/mixins/_mixins.scss]: done

File: packages/styles/src/textarea.scss (hunk 2 above)
Comment (droshev) [reply packages/styles/src/textarea.scss]: this was a mistake, thanks

**Pull request merged**